app/src/main/java/net/kuama/documentscanner/presentation/CropperActivity.kt

```diff
@@ -24,6 +24,7 @@ import java.util.*
 class CropperActivity : AppCompatActivity() {
     private lateinit var cropModel: CropperModel
     private lateinit var bitmapUri: Uri
+    private var screenOrientationDeg: Int? = 0
```

Comment:
```suggestion
    private var screenOrientationDeg: Int = 0
```
or
```suggestion
    private var screenOrientationDeg: Int?
```
choose nullable or with default value